src/common/ConfirmationDialogue.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from '@mui/material/Button'
+import Dialog from '@mui/material/Dialog'
+import DialogTitle from '@mui/material/DialogTitle'
+import Box from '@mui/material/DialogContentText'
+import DialogActions from '@mui/material/DialogActions'
+import Typography from '@mui/material/Typography'
+import DialogContent from '@mui/material/DialogContent'
+
+export default function ConfirmationDialogue({
+  showConfirmationDialogue,
+  confirmationModalTitle,
+  confirmationContent,
+  onCancel,
+  onConfirm,
+}: {
+  showConfirmationDialogue: boolean
+  confirmationModalTitle: string
+  confirmationContent: JSX.Element
+
+  onCancel: () => void
+  onConfirm: () => void
+}) {
+  return (
+    <Dialog open={showConfirmationDialogue} onClose={onCancel}>
+      <DialogTitle id='alert-dialog-title'>{confirmationModalTitle}</DialogTitle>
+      <DialogContent>
+        <Typography>Are you sure you want to perform this action?</Typography>
+      </DialogContent>
+      {confirmationContent}
```

Comment:
Should this not be inside `DialogContent`? You can wrap it in a `Box` to allow you to put some padding/margin between it and the other content

Reply:
If this is changed to be a string as recommended in https://github.com/gchq/Bailo/pull/141/files#r975493106 then I'd suggest the following changes here:

```
      <DialogContent>
        <Typography>Are you sure you want to perform this action?</Typography>
        <Alert severity='error' sx={{ m: 2, width: 400 }}>
          {errorMessage}
        </Alert>
      </DialogContent>
```